<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CcdTab</class>
 <widget class="QWidget" name="CcdTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>704</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxPic">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>270</height>
      </size>
     </property>
     <property name="title">
      <string>图像</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxBinPic">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>270</height>
      </size>
     </property>
     <property name="title">
      <string>二值化图像</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>144</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>设置</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>图像宽度：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>图像高度：</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxWidth">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxHeight">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>结束指令：</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditEndCode">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonNoZoom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>89</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>不缩放</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAutoZoom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>自动缩放</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>171</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>数据</string>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>最大值：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>最小值：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>平均值：</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>中间值：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMaxValue">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelMinValue">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelAverage">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>61</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelMidValue">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelPixelValue">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>像素值：</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelCursorPos">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>光标位置：</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="minimumSize">
        <size>
         <width>201</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>201</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>二值化</string>
       </property>
       <widget class="QRadioButton" name="radioButton_mid">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>中间值作为阀值</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_fixed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>固定值作为阀值</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxTh">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderTh">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_custom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>自定义算法</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_openDll">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>选择算法</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_dllName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>100</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonSaveData">
          <property name="text">
           <string>保存数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSavePicture">
          <property name="text">
           <string>保存图像</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
